.github/workflows/build_and_test_debug.yml

```diff
@@ -114,16 +114,53 @@ jobs:
       - name: Set XCode Version
         run: sudo xcode-select -switch /Applications/Xcode_13.2.app
 
-      - name: Build OSX Client
+      - name: Build MacOS Client
         run: npm run action cordova/build osx
```

Comment:
i'm partially inclined to rename the `osx` platform to `macos`

Reply:
Yes, let's use the modern names. We could use macos for the commands too, perhaps with osx as an alias.